lib/anima/attribute.rb

```diff
@@ -42,7 +42,7 @@ def initialize(name)
     #
     def load(object, attributes)
       value = attributes.fetch(name) do |attribute_name|
-        fail Error::Missing.new(object, attribute_name)
+        return
```

Comment:
I doubt the whole block is needed, as we guarded the existence of all keys already with your addition. I think `value = attributes.fetch(name)` has the same semantics.